pkg/network/vmispec/hotplug.go

```diff
@@ -32,10 +36,39 @@ func NetworksToHotplug(networks []v1.Network, interfaceStatus []v1.VirtualMachin
 	return networksToHotplug
 }
 
-func indexedInterfacesFromStatus(interfaces []v1.VirtualMachineInstanceNetworkInterface) map[string]v1.VirtualMachineInstanceNetworkInterface {
+func indexedInterfacesFromStatus(interfaces []v1.VirtualMachineInstanceNetworkInterface, predicates ...func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool) map[string]v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
I want the exact opposite behavior - a simple or. 

Either predicate works, your stuff goes into the filtered slice.

Reply:
Well, I do not understand why we need it.
I do not see anything using more than one at the moment.
And if anyone will need something like what you mentioned, then they can embed it into one predicate without the need for many (which complicates things).

Reply:
I'm using variadic input to have an *optional* predicate in the function - some callers don't need to pass a predicate, they just want the indexed interfaces. 

I prefer it to be optional because it feels silly to pass it an "always true" predicate - that was my original code. 

... I guess I was trying to be clever, which might not have been the best alternative.

I'm willing to compromise, but I still think there's literally nothing wrong with the code as it was before. 

Would you prefer that? (Using an always true predicate)

Reply:
I think this was resolved